training/distributed_training/pytorch/data_parallel/yolov5/yolov5.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    print(f"SageMaker Execution Role:{role}")
</pre><p>Style nit: add a space between colon and string substitution for better readability of the print statement, ie <code>Role:{role}</code> -&gt; <code>Role: {role}</code>  . At least 3 occurrences in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>